bindings/c/test/client_config_tester.cpp

```diff
@@ -208,12 +224,23 @@ bool parseArgs(int argc, char** argv) {
 	return true;
 }
 
+void exitImmediately(int exitCode) {
+#ifdef _WIN32
+	// This function is documented as being asynchronous, but we suspect it might actually be synchronous in the
+	// case that it is passed a handle to the current process. If not, then there may be cases where we escalate
+	// to the crashAndDie call below.
```

Comment:
I think this comment may not be completely applicable here

Reply:
True, I will remove it